content/blog/2024-05-16.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Version 2.0.0 released!"
+date: 2024-05-15T11:00:00+02:00
+description: "Version 2.0.0 released!"
+draft: false
+---
+
+## It's time to celebrate the 2.0.0 release of Kaoto
+
+With this 2.0.0 release we finished implementing the foundational functionalities for an [Apache Camel](https://camel.apache.org/) visual editor. Kaoto will guide you with the creation of new and the editing of existing [Apache Camel](https://camel.apache.org/) integrations. Together with the other extensions of the [Camel Extension Pack](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack) it offers functionality from the creation over the local execution with the [Camel CLI](https://camel.apache.org/manual/camel-jbang.html) to even the deployment to your cluster.
+
+This release is just the beginning for Kaoto! We have [so many exciting features and improvements](https://github.com/orgs/KaotoIO/projects/14/views/2) in the works, and we're eagerly inviting anyone who wants to join us on this journey. Let's all hop on this Camel ride together! 🐫🎉
+
+## Give it a try!
+
+* Kaoto [quickstart](/docs/quickstart/).
+* Kaoto is available as a [VSCode extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto).
+* Kaoto [showcase deployment](https://red.ht/kaoto).
+
+## Features & Improvements
```

Comment:
I think we should include new features from TP3.1 https://github.com/KaotoIO/kaoto/releases/tag/2.0.0-TP3.1 as the last blogpost was for TP3

Reply:
fixed